modules/DoneAction.js

```diff
@@ -5,6 +5,7 @@ const actions = {
         "Sleep": ["rundll32.exe", "powrprof.dll,SetSuspendState", "0,1,0"],
         "Lock": ["rundll32.exe", "user32.dll,LockWorkStation"],
         "Shutdown": ["shutdown", "/s", "/f", "/t", "0"],
+        "Hibernate": ["shutdown", "/h", "/f", "/t", "0"],
```

Comment:
https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/shutdown

Puts the local computer into hibernation, if hibernation is enabled. The /f switch can be used with the /h switch.


Puts the local computer into hibernation, if hibernation is enabled. The /f switch can be used with the /h switch.


<br class="Apple-interchange-newline" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none;">


Puts the local computer into hibernation, if hibernation is enabled. The /f switch can be used with the /h switch.


<br class="Apple-interchange-newline" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none;">



Puts the local computer into hibernation, if hibernation is enabled. The /f switch can be used with the /h switch.


<br class="Apple-interchange-newline" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none;">



Puts the local computer into hibernation, if hibernation is enabled. The /f switch can be used with the /h switch.


<br class="Apple-interchange-newline" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none;">



Puts the local computer into hibernation, if hibernation is enabled. The /f switch can be used with the /h switch.


<br class="Apple-interchange-newline" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none;">


Puts the local computer into hibernation, if hibernation is enabled. The /f switch can be used with the /h switch.


<br class="Apple-interchange-newline" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none;">

Reply:
```suggestion
        "Hibernate (if enabled)": ["shutdown", "/h", "/f", "/t", "0"],
```

Reply:
Looks like this:

![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/36509607/cd047182-a55e-4420-a666-80583ba56317)

Reply:
You are right
![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/11440331/00a5a49d-45c1-4986-85ee-b077fc7cfaaf)